Packs/Carbon_Black_Enterprise_Response/Integrations/CarbonBlackResponseV2/CarbonBlackResponseV2.py

```diff
@@ -861,6 +861,15 @@ def fetch_incidents(client: Client, max_results: int, last_run: dict, first_fetc
         if not alert_id or not alert_name:
             demisto.debug(f'{INTEGRATION_NAME} - Alert details are missing. {str(alert)}')
 
+        if ioc_attr := alert.get('ioc_attr'):
+            try:
+                alert['ioc_attr'] = json.loads(ioc_attr)
+                highlights = alert['ioc_attr'].get('highlights', {})
+                for i, attribute in enumerate(highlights):
+                    highlights[i] = attribute.replace("PREPREPRE", "").replace("POSTPOSTPOST", "")
+            except Exception:
+                pass
```

Comment:
if we need the `json.loads` and want to catch an error in parsing the JSON you can catch it with JSONDecodeError and then please add demisto.debug to describe the error,
for example:
        except JSONDecodeError as e:
            demisto.debug(f"Failed to parse ioc_attr as JSON: {e}")